src/python/pants/backend/scala/subsystems/scalac.py

```diff
@@ -20,9 +20,7 @@ class Scalac(Subsystem):
         "scalac_plugins.default.lockfile.txt",
     )
 
-    args = ArgsListOption(
-        help=f"Global `scalac` compiler flags, e.g. `--{options_scope}-args='-encoding UTF-8'`."
-    )
+    args = ArgsListOption(example="-encoding UTF-8")
```

Comment:
Notable difference.
Before: `Global `scalac` compiler flags, ...`
After: `Arguments to pass directly to scalac, ...`